openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -115,7 +115,8 @@ <h3 class="message__title">{% trans "We could not save your progress" %}</h3>
                             </li>
                             {% if allow_latex %}
                                 <li class="list--actions__item">
-                                    <button type="submit" class="submission__preview  action action--save" aria-describedby="response__preview_explanation__{{ xblock_id }}">
+                                    <button type="submit" class="submission__preview  action action--save" aria-describedby="response__preview_explanation__{{ xblock_id }}"
```

Comment:
Once I fixed the test, it failed because the Preview button was enabled upon initial rendering. The logic about when it should be enabled is a bit funky, but I decided that for initial rendering, going with whether or not there is something to Submit is good enough.